src/nginxconfig/util/vue_backwards_compatibility.js

```diff
@@ -26,6 +26,72 @@ THE SOFTWARE.
 
 import isObject from './is_object';
 import deepMerge from './deep_merge';
+import { accessLogPathDefault, accessLogParamsDefault, errorLogPathDefault, errorLogLevelDefault } from './logging';
+import { serverDomainDefault } from './defaults';
+
+// Migrate old logging settings to new ones
+const migrateLogging = data => {
+    if (Object.keys(data).length === 0) {
+        // Not having anything to migrate, so skip this logic
+    }
+
+    const globalLogging = 'logging' in data.global && isObject(data.global.logging) ? data.global.logging : {};
+
+    // global access_log
+    const [globalAccessLogPath, ...globalAccessLogParameters] = (globalLogging.accessLog || accessLogPathDefault).split(' ');
+    const globalAccessLogEnabled = 
+        !('accessLog' in globalLogging) || // accessLog was enabled by default and might not appear at all
+        (globalAccessLogPath !== '' && globalAccessLogPath !== 'off'); // *or* someone turned it off explicitly
+
+    // global error_log
+    const [globalErrorLogPath, ...globalErrorLogLevel] = (globalLogging.errorLog || `${errorLogPathDefault} ${errorLogLevelDefault}`).split(' ');
+    const globalErrorLogEnabled = 
+        !('errorLog' in globalLogging) || // errorLog was enabled by default and might not appear at all
+        globalErrorLogPath !== '/dev/null'; // *or* someone turned it off explicitly
```

Comment:
```suggestion
        (globalErrorLogPath !== '' && globalErrorLogPath !== '/dev/null'); // *or* someone turned it off explicitly
```
per the current tool's behavior, the query param is set to an empty string when off